install_config/install/disconnected_install.adoc

```diff
@@ -134,8 +139,9 @@ attach it:
 yum repositories, and `createrepo` can create a usable `yum` repository from a
 directory:
 +
+[source, bash]
 ----
-# yum -y install yum-utils createrepo docker git
+$ yum -y install yum-utils createrepo docker git
```

Comment:
@gaurav-nelson I think this needs a `sudo` in front if it's not going to be run as root user.